backend/apps/matching/models/user.py

```diff
@@ -34,7 +35,7 @@ def new(email, pwd=None, **kwargs):
         if pwd is not None:
             user.set_password(pwd)
         else:
-            user.set_password(email)
+            user.set_password(get_random_string(length=32))
```

Comment:
I think we should switch to `def new(email, pwd, **kwargs)` and don't have any process of setting an automatic password.

Reply:
Sure